src/BenchmarkDotNet/Environments/Runtimes/WasmRuntime.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using BenchmarkDotNet.Jobs;
+
+namespace BenchmarkDotNet.Environments
+{
+    public class WasmRuntime : Runtime, IEquatable<WasmRuntime>
+    {
+        public static readonly WasmRuntime Default = new WasmRuntime("Wasm");
+
+        public WasmRuntime(string name) : base(RuntimeMoniker.Wasm, "wasm", name)
```

Comment:
the moniker should be `net5.0` instead of `wasm` ?

```suggestion
        public WasmRuntime(string name) : base(RuntimeMoniker.Wasm, "net5.0", name)
```